java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -73,7 +73,12 @@ public Table(ColumnVector... columns) {
     nativeHandle = createCudfTableView(viewPointers);
   }
 
-  private Table(long[] cudfColumns) {
+  /**
+   * Table class makes a copy of the array of cudfColumns passed to it. The class will decrease the
+   * refcount on itself and all its contents when closed and free resources if refcount is zero
+   * @param cudfColumns - Array of nativeHandles
+   */
+  public Table(long[] cudfColumns) {
```

Comment:
@revans2, Please let me know if it's okay to change it to public.

Reply:
public is not okay.  It should be package private

Reply:
Thanks. Made it package private.